<launch>

    <node pkg="contact_stability" type="static_server.py" name="static_stability" output="screen" />

</launch>
